ADMIN.adoc

```diff
@@ -0,0 +1,36 @@
+= Repository administrator responsibilities
+
+== Administrator rotation schedule
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be an administrator is voluntary.
+To volunteer to be an administrator, contact @cbudz or @bergerhoffer.
+
+== Administrator responsibilities
+
+As the administrator, you are the point of contact if any contributors experience issues with the repository.
+You also have the following responsibilities:
+
+* Driving resolution of open issues
+* Driving completion of pull requests
+* Maintaining the Style Guide site on GitHub
+
+=== Driving resolution of open issues
+
+* Review link:https://github.com/redhat-documentation/supplementary-style-guide/issues[issues] submitted to the supplementary style guide GitHub repository.
+* If none of the project members respond to a newly opened issue, add the _Needs Fix_ label to the issue and mention @redhat-documentation/ccs-style-council in a comment on the issue to notify project members about it.
```

Comment:
> I would also recommend updating the instruction to flag the issue with "Needs Fix" (we can add that if people agree) and simply @ the admin group (redhat-documentation/ccs-style-council)

@cbudz I added a note about the _Needs Fix_ label, but was not sure where you wanted me to place it. Is it OK if I include it under _Driving resolution of open issues_?

Reply:
@inoxx03 That does look like a logical place to me.

Reply:
@inoxx03 that works for me.